pkg/ovs/openflow/testing/openflow_test_utils.go

```diff
@@ -99,7 +104,7 @@ func OfctlDumpFlows(brName string, table uint8) ([]string, error) {
 }
 
 func runOfctlCmd(cmd, brName string, table uint8) ([]byte, error) {
-	cmdStr := fmt.Sprintf("sudo /usr/bin/ovs-ofctl -O Openflow13 %s %s table=%d", cmd, brName, table)
+	cmdStr := fmt.Sprintf("ovs-ofctl -O Openflow13 %s %s table=%d", cmd, brName, table)
```

Comment:
Out of curiosity, why some commands need sudo and some don't?

Reply:
I don't think any of the commands should use sudo. The integration tests are already supposed to be run as root.

Reply:
OK, I will remove all "sudo" here.